### ISSUE TITLE
What is 'remainder'?

### ISSUE THREAD

**Issue by youngledo** [eclipse-mat/mat#73]
I see that 'remainder' occupies a certain size on the interface, but I cannot see what it actually has. Can you explain what it is and when it will appear?

![image](https://github.com/user-attachments/assets/29147ff6-68da-4ba9-b4e0-530d1779b012)

At times, its proportion is still significant, but I cannot see why it is so large.
<img width="835" alt="企业微信截图_29cfc2b7-8d4a-48b7-ab9b-9f7382d9851b" src="https://github.com/user-attachments/assets/e4ab48ca-31c8-48fc-861c-e790978e2cbd">

**Comment by youngledo** [comment]
Thanks, isn't this remainder also part of the heap, don't you need to pay attention to this memory?

**Comment by kgibm** [comment]
> Thanks, isn't this remainder also part of the heap, don't you need to pay attention to this memory?

Yes. The remaining dominators didn't exceed 1% of the heap so generally in this case the recommendation is to review other queries in MAT: https://help.eclipse.org/latest/topic/org.eclipse.mat.ui.help/gettingstarted/basictutorial.html

If none of those core queries help, generally what I do then is to go to the Histogram, sort by retained heap, find the largest non-JDK & non-framework classes, right click, Merge Shortest Paths to GC roots excluding phantom/soft/weak/etc. references and then calculate the minimum retained heap for referenced objects and explore the shortest paths to GC roots that retains the most of these.
